plugin/src/main/java/com/playmonumenta/scriptedquests/timers/CommandTimer.java

```diff
@@ -46,13 +58,18 @@ public void run() {
 					return;
 				}
 
-				for (CommandTimerInstance timer : mTimers.values()) {
+				for (CommandTimerInstance timer : mTickTimers.get(mCounter).values()) {
 					timer.tick(plugin);
 				}
+				mCounter++;
+				if (mCounter >= mPeriod) {
+					mCounter = 0;
+				}
+
 			}
 		};
```

Comment:
Can we move this whole runnable to a function that can be called to tick? Before it was nice that each one had a runnable at the appropriate frequency, but now all of these are going to be running every tick, we might as well make it a function directly instead of a scheduled task per timer period. Then we can add a runnable in the timer manager to call all of those functions every tick.